src/scripts/bootstrap_aws_context_creds.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+function run_main() {
+    rm -f ~/.aws/credentials
+    mkdir -p ~/.aws
+    set | grep ^CTX_AWS_BASE_64_CREDS_ | cut -d= -f2- | while read -r base64creds; do \
+        echo "$base64creds" | base64 -d >> ~/.aws/credentials; \
+    done;
+}
+
+function get_line() {
+    read -r line < ~/.aws/credentials
+    echo "$line"
+}
```

Comment:
Didn't know how to do this in the batch script, but this code can be easily migrated out of here.

Reply:
You should be able to replace this directly with a call to `head -n 1 ~/.aws/credentials`